web/src/app/core/services/repository.service.ts

```diff
@@ -51,4 +51,27 @@ export class RepositoryService {
     const callable: any = this.fns.httpsCallable('deleteGitWebhookRepository');
     return callable({ repositoryUid, token: this.authService.profile.oauth.githubToken });
   }
+
+  public getRepoRating(repo: RepositoryModel): number {
+    let rating: number = 0;
+    const issuePoints: number = repo.issues.length > 0 ? this.getPoints(repo.issues[0].createdOn) : 0;
+    const releasesPoints: number = repo.releases.length > 0 ? this.getPoints(repo.releases[0].createdOn) : 0;
+    rating = (issuePoints + releasesPoints) * 5;
```

Comment:
Need to calculate percentage, so when more rating calculations are added, it is always out of 100%.

I think here should be divide by the amount (`2` at the moment), and below `getPoints` returns a percentage out of `100`

Reply:
done